src/Streamly/Internal/System/Command.hs

```diff
@@ -101,15 +105,15 @@ quotedWord =
 -- /Internal/
 {-# INLINE streamWith #-}
 streamWith :: MonadCatch m =>
-    (FilePath -> [String] -> SerialT m a) -> String -> SerialT m a
+    (FilePath -> [String] -> Stream m a) -> String -> Stream m a
 streamWith f cmd =
-    Stream.concatM $ do
-        xs <- Stream.toList
+    Stream.concatMapM (\() -> do
```

Comment:
This was a little weird to write. `concatM`.

Reply:
We should have it in Interal.Data.Stream. And maybe should call it concatEffect.